### PR TITLE
Testing/cypress setup

### PR DESCRIPTION
**Pull request by mattsaqueli**
# Files changed:
All Cypress test files.
# Change Summary:
Tested the dashboard and favorites page.
# Additional Questions/Concerns:
N/A - Next, test the About Us and Resources page, then deploy using Vercel.

### CODE REVIEW

File: cypress/e2e/app_spec.cy.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+describe('App', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.themotivate365.com/stoic-quote', {
+      statusCode: 200,
+      fixture: 'quote.json'
+    }).as('getQuote')
```
Comment (hjawad22) [inline cypress/e2e/app_spec.cy.js:6]: I see you used an alias here but I do not see it in your code, you may not need this here and throughout your code.

File: cypress/e2e/app_spec.cy.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+describe('App', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.themotivate365.com/stoic-quote', {
+      statusCode: 200,
+      fixture: 'quote.json'
```
Comment (hjawad22) [inline cypress/e2e/app_spec.cy.js:5]: you can just use `quote` for the fixture here, no need foe the `.json`.

File: cypress/e2e/app_spec.cy.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }).as('getQuote')
+
+    cy.visit('http://localhost:3000/')
+  })
+
+  it('should have navigation links', () => {
+    cy.get('.nav-section')
+      cy.contains('Home').should('exist')
```
Comment (hjawad22) [inline cypress/e2e/app_spec.cy.js:13]: I do not believe you need the cy after each contains, you can just use `.contains` here and throughout your code.

File: cypress/e2e/app_spec.cy.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      cy.contains('About Us').click()
+      cy.url().should('include', '/about')
+      
+      cy.contains('Resources').click()
+      cy.url().should('include', '/resources')
+  })
+
+  it('should display a quote on the home page', () => {
+    cy.get('.quote-section').should('contain', 'True will is quiet humility, resilience, and flexibility; the other kind of will is weakness disguised by bluster and ambition.')
+  })
```
Comment (hjawad22) [inline cypress/e2e/app_spec.cy.js:36]: This is a great start, but if you have multiple quotes I would test for the amount you are expecting to see on the DOM.

File: cypress/e2e/favorites_spec.cy.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+describe('Favorites', () => {
+  beforeEach(() => {
+    cy.fixture('favquote.json').then(quote => {
+      cy.intercept('GET', 'https://api.themotivate365.com/stoic-quote', {
```
Comment (hjawad22) [inline cypress/e2e/favorites_spec.cy.js:4]: Is this a post-request intercept? I am seeing two 'GET' intercepts.

File: cypress/e2e/app_spec.cy.js (hunk 1 above)
Comment (mattsaqueli) [reply cypress/e2e/app_spec.cy.js:6]: Yes thank you for catching this! I began to use it but decided not to and forgot to remove it.

**Pull request merged**